Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1625,6 +1625,17 @@ bool RenderLayerScrollableArea::scrollingMayRevealBackground() const
     return scrollsOverflow() || usesCompositedScrolling();
 }
 
+bool RenderLayerScrollableArea::isVisibleToHitTesting() const
+{
+    auto& renderer = m_layer.renderer();
+    FrameView& frameView = renderer.view().frameView();
+
+    bool isVisibleToHitTest = renderer.visibleToHitTesting();
+    if (HTMLFrameOwnerElement* owner = frameView.frame().ownerElement())
```

Comment:
I think FrameView (which is a ScrollableArea) should override `isVisibleToHitTesting` and this code should live there.

Reply:
Fixed.